test/higherorderfns.jl

```diff
@@ -709,24 +709,43 @@ end
     @test extrema(f, x) == extrema(f, y)
     @test extrema(spzeros(n, n)) == (0.0, 0.0)
     @test extrema(spzeros(n)) == (0.0, 0.0)
-    @test_throws ArgumentError extrema(spzeros(0, 0))
-    @test_throws ArgumentError extrema(spzeros(0))
+    # @test_throws "reducing over an empty" extrema(spzeros(0, 0))
+    # @test_throws "reducing over an empty" extrema(spzeros(0))
     @test extrema(sparse(ones(n, n))) == (1.0, 1.0)
     @test extrema(sparse(ones(n))) == (1.0, 1.0)
     @test extrema(A; dims=:) == extrema(B; dims=:)
     @test extrema(A; dims=1) == extrema(B; dims=1)
     @test extrema(A; dims=2) == extrema(B; dims=2)
     @test extrema(A; dims=(1,2)) == extrema(B; dims=(1,2))
     @test extrema(f, A; dims=1) == extrema(f, B; dims=1)
-    @test extrema(sparse(C); dims=1) == extrema(C; dims=1)
+    # @test_throws "reducing over an empty" extrema(sparse(C); dims=1) == extrema(C; dims=1)
```

Comment:
Does this work after https://github.com/JuliaLang/julia/pull/43604?

```suggestion
    # TODO: enable the following once https://github.com/JuliaLang/julia/pull/43604 is merged
    @test_broken "reducing over an empty" extrema(sparse(C); dims=1) == extrema(C; dims=1)
```